docs/build_version_doc/build_all_version.sh

```diff
@@ -68,6 +68,7 @@ for tag in $tag_list; do
           git checkout master -- docs/mxdoc.py
     fi
     git submodule update || exit 1
+    git submodule update --init --recursive
```

Comment:
Why not update line 70?

Reply:
I have tried updating it together, but its unstable. I have found this to be better and working consistently